runtime/autoload/man.vim

```diff
@@ -56,20 +56,19 @@ function! man#open_page(count, count1, ...) abort
   endtry
   call s:push_tag()
   let bufname = 'man://'.name.(empty(sect)?'':'('.sect.')')
-  let found_man = s:find_man()
-  if getbufvar(bufname, 'manwidth') ==# s:manwidth()
-    if found_man
+  if s:find_man()
```

Comment:
I changed it to work just like `:help`. `:help` does not look for the closest window only if `tab` is in the modifiers.